packages/x-components/src/x-modules/history-queries/components/my-history.vue

```diff
@@ -33,10 +33,13 @@
                   @binding {Suggestion} suggestion - History Query suggestion data
                   @binding {number} index - History Query suggestion index
             -->
-                <slot name="suggestion-content" v-bind="{ suggestion, index }">
-                  <div class="x-list x-list--vertical">
-                    <span>{{ suggestion.query }}</span>
-                    <span>{{ formatTime(suggestion.timestamp) }}</span>
+                <slot name="suggestion-content" v-bind="{ suggestion, index, formatTime }">
+                  <div class="x-list x-list--horizontal">
+                    <HistoryIcon />
```

Comment:
I'd just bind the `formatTime` function to the slot and avoid adding the icon here in x. I'd leave that to the `archetype`, where we are usually customising the icons and styles. What do you think?